vignettes/OrganismDbi.Rmd

```diff
@@ -0,0 +1,190 @@
+---
+title: "OrganismDbi: A meta framework for Annotation Packages"
+author:
+- name: "Marc Carlson"
+- name: "Aliyu Atiku Mustapha"
+  affiliation: "Vignette translation from Sweave to Rmarkdown / HTML."
+date: "`r format(Sys.time(), '%B %d, %Y')`"
+package: OrganismDbi
+vignette: >
+    %\VignetteEngine{knitr::rmarkdown}
+    %\VignetteIndexEntry{OrganismDbi: A meta framework for Annotation Packages}
+    %\VignetteEncoding{UTF-8}
+    %\VignetteKeywords{OrganismDbi}
+    %\VignettePackage{OrganismDbi}
+output:
+  BiocStyle::html_document:
+  numbered_sections: true
+  toc: true
+  toc_depth: 4
+---
+
+`r Biocpkg("OrganismDbi")` is a software package that helps tie together
+different annotation resources. It is expected that users may have previously
+made or seen packages like `r Biocpkg("org.Hs.eg.db")` and 
+`r Biocpkg("TxDb.Hsapiens.UCSC.hg19.knownGene")`. Packages like these two are
+very
+different and contain very different kinds of information, but are still about
+the same organism: Homo sapiens. The `r Biocpkg("OrganismDbi")` package allows
+us to combine resources like these together into a single package resource,
+which can represent ALL of these resources at the same time. An example of this
+is the `r Biocpkg("Homo.sapiens")` package, which combines access to the two
+resources above along with others.
+
+This is made possible because the packages that are represented by 
+`r Biocpkg("Homo.sapiens")`
+are related to each other via foreign keys.
```

Comment:
Could you fix the formatting for the 2 paragraphs in line 22-36 in RStudio via `Code > Reflow Comment`?